public/app/plugins/datasource/prometheus/querybuilder/components/promQail/PromQail.tsx

```diff
@@ -33,6 +33,20 @@ export const PromQail = (props: PromQailProps) => {
 
   const [state, dispatch] = useReducer(stateSlice.reducer, initialState(query, !skipStartingMessage));
 
+  const [labelNames, setLabelNames] = useState([])
```

Comment:
Could we add more typing here?
```suggestion
  const [labelNames, setLabelNames] = useState<string[]>([])
```

Fixes this warning on L45
```
Argument of type 'string[]' is not assignable to parameter of type 'SetStateAction<never[]>'.
  Type 'string[]' is not assignable to type 'never[]'.
    Type 'string' is not assignable to type 'never'.
```